### ISSUE TITLE
[FEATURE] Create theme setting page

### ISSUE THREAD

**Issue by BhartiNagpure** [narayan954/dummygram#736]
### Description

I would like to create theme setting page. 

### Screenshots

![Screenshot 2023-07-10 155832](https://github.com/narayan954/dummygram/assets/104631882/045e8a69-4f6c-4dbb-bd2e-485621e9bec8)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
This background theme colour looks so nice, as well as the issue. You can work on this! :)

Also can you please provide the colour of background? I'm thinking to change our application's colour to it

**Comment by narayan954** [comment]
> This background theme colour looks so nice, as well as the issue. You can work on this! :)
> 
> Also can you please provide the colour of background? I'm thinking to change our application's colour to it

Also thinking of changing button colours to this as well. Can you please provide that colour as well. Thanks a lot!
Also what're your opinions on this??

**Comment by BhartiNagpure** [comment]
> > This background theme colour looks so nice, as well as the issue. You can work on this! :)
> > Also can you please provide the colour of background? I'm thinking to change our application's colour to it
> 
> Also thinking of changing button colours to this as well. Can you please provide that colour as well. Thanks a lot! Also what're your opinions on this??

Are u saying to change all  button color of website in to sky blue color which I have provided?

**Comment by narayan954** [comment]
> > > This background theme colour looks so nice, as well as the issue. You can work on this! :)
> > > Also can you please provide the colour of background? I'm thinking to change our application's colour to it
> > 
> > 
> > Also thinking of changing button colours to this as well. Can you please provide that colour as well. Thanks a lot! Also what're your opinions on this??
> 
> Are u saying to change all button color of website in to sky blue color which I have provided?

yes, basically along with a different colour of darkmode background. But that's a separate issue :)

**Comment by BhartiNagpure** [comment]
Actually I am trying to create theme setting page I'm set the route but still it  shows 404 page

**Comment by BhartiNagpure** [comment]
> I was thinking to change to this colour theme instead of the pink theme and neons in our web application

Ok I will try to make this changes asap.

**Comment by narayan954** [comment]
> Actually I am trying to create theme setting page I'm set the route but still it shows 404 page


is that 404 page of our application or github??
If github, it could be because of github pages? [try this out](https://stackoverflow.com/questions/71984401/react-router-not-working-with-github-pages)

**Comment by BhartiNagpure** [comment]
Can you tell mi if I would like to create theme setting page in how many page I want to add that themesetting .

**Comment by narayan954** [comment]
> Can you tell mi if I would like to create theme setting page in how many page I want to add that themesetting .

I think each page for each theme section should do. Like for the theme settings, account settings, sound/notification settings. You may add them as a placeholder for now, they'll implemented incrementally

**Comment by BhartiNagpure** [comment]
@narayan954  can you help me to route on theme page 
I'm making changes but still it shows 404 page of app.

**Comment by narayan954** [comment]
> @narayan954 can you help me to route on theme page I'm making changes but still it shows 404 page of app.

Yeah sure, how can I help? Discord?
